qutebrowser/utils/version.py

```diff
@@ -449,8 +454,12 @@ def version():
         lines += ['{}: {}'.format(name, path)]
 
     lines += [
-        '',
-        'Uptime: {}'.format(_uptime()),
+        "",
+        "Autoconfig loaded: {}".format(_autoconfig_loaded())
+    ]
+
+    lines += [
+        'Uptime: {}'.format(_uptime())
     ]
```

Comment:
Please unify those two `lines += [ ... ]` blocks into one (similar to the blocks above).